src/StringParser/AbstractStringParser.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Noodlehaus\StringParser;
+
+/**
+ * Abstract string parser
+ *
+ * @package    Config
+ * @author     Jesus A. Domingo <jesus.domingo@gmail.com>
```

Comment:
Please, add yourself as author of this class, removing the others.

Reply:
I've fixed this.